commandline/WSL/faq.md

```diff
@@ -122,6 +123,28 @@ Mount points for hard drives on the local machine are automatically created and
  
 Example usage would be `cd /mnt/c` to access c:\
 
+### How do I use a Windows file with a Linux app?
+
+One of the benefits of WSL is being able to use the same file with both Windows and Linux apps or tools.
+
+One of the main limitations of using WSL is that changing Linux files with a Windows app or tool is not allowed. See: [Do not change Linux files using Windows apps and tools](https://blogs.msdn.microsoft.com/commandline/2016/11/17/do-not-change-linux-files-using-windows-apps-and-tools/)
+
+However changing Windows files with a Linux app or tools is allowed.
+
+One way to use a Windows file with a Linux app is to use an absolute path, e.g. `/mnt/c/Users/<Windows User>/Documents/Projectes/<filename>`.
+
+However not all Linux apps or tools can access a file using `/mnt`.
+
+A solution is to create a [symbolic link](http://manpages.ubuntu.com/manpages/precise/man7/symlink.7.html).

```

Comment:
@bitcrazed 

Any changes you wish to make are welcome and appreciated. I just wanted a section in the WSL FAQ that noted the reason for needing to use symlinks and how to make one. It doesn't matter to me by who or how it gets done. :)

Do you need me to make the changes and resubmit or will you just make them and apply the change?